tests/blob/enc/blob.txt

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What are the constaints about what the blob can/should contain? It seems like comments are legal. Is it just up to enclave writer to throw whatever they want in here and use it as they wish or should OE expect some format?